app/views/idv/finance/new.html.slim

```diff
@@ -6,10 +6,21 @@ p.italic Please provide only one of the following:
 = simple_form_for(idv_finance_form, url: idv_finance_path,
     html: { autocomplete: 'off', method: 'put', role: 'form' }) do |f|
   = f.error_notification
-  = f.collection_radio_buttons :finance_type, Idv::FinanceForm.finance_type_choices,
-    :first, :last, required: true, item_wrapper_tag: :div
-  = f.input :finance_account, label: false
+  .mb3.js-finance-choice-cntnr
+    = f.collection_radio_buttons(:finance_type, Idv::FinanceForm.finance_type_choices,
+      :first, :last, required: true, item_wrapper_tag: :div) do |r|
+      = r.label(class: 'radio')
+        = r.radio_button
+        span.indicator
+        = r.text
 
+  - label_txt = t('idv.form.finance_unselected')
+  - Idv::FinanceForm::FINANCE_TYPES.each do |finance_type|
+    - if idv_params[finance_type].present?
+      - label_txt = t("idv.form.#{finance_type}")
```

Comment:
These 4 lines can be reduced to 2 lines:
```slim
- finance_type = Idv::FinanceForm::FINANCE_TYPES.find { |ft| ft == idv_params[ft] } || 'finance_unselected'
- label_txt = t("idv.form.#{finance_type}")
```
And they can then be extracted to a decorator to keep logic out of views:
```ruby
FinanceFormDecorator = Struct.new(:idv_params) do
  def label_text
    I18n.t("idv.form.#{finance_type}")
  end

  private

  def finance_type
    finance_types.find { |ft| ft == idv_params[ft] } || 'finance_unselected'
  end

  def finance_types
    Idv::FinanceForm::FINANCE_TYPES
  end
end
```
And in the finance controller:
```ruby
def new
  @label_text = FinanceFormDecorator.new(idv_params).label_text
 end
```
Then in the view:
```slim
...
= f.input :finance_account, required: true,
    label: @label_text, label_html: { class: 'js-finance-label' }
...
```